sdk/Sdk.Analyzers/Extensions/TypeSymbolExtensions.cs

```diff
@@ -58,6 +58,11 @@ internal static string GetMinimalDisplayName(this ITypeSymbol type, SemanticMode
                 name = Regex.Match(name, @"IEnumerable<[^>]+>").Value;
             }
 
+            if (name.Contains("System."))
```

Comment:
double checking that we want `Contains()` here and not `Startswith("System.")`

Reply:
Either work I think but I like the idea of Startswith